apps/web/src/pages/templates/hooks/usePreviewPushTemplate.ts

```diff
@@ -47,11 +44,11 @@ export const usePreviewPushTemplate = (locale?: string) => {
   useEffect(() => {
     getPushPreview({
       locale,
-      content: previewData.current.templateContent,
-      payload: previewData.current.processedVariables,
-      title: previewData.current.templateTitle,
+      content: templateContent,
+      payload: processedVariables,
```

Comment:
In the getPushPreview implementation(in the mutateAsync call) - you need to make sure to parse the payload : `payload: JSON.parse(payload)`
as processedVariables is a string